heketi-centos-ci-tests.sh

```diff
@@ -39,7 +39,9 @@ yum -y install \
 	python-virtualenv \
 	ansible \
 	libvirt \
-	libvirt-devel
+	libvirt-devel \
+        ruby \
```

Comment:
Not a biggie as this is shell, but the indent here is different than the lines above?

Reply:
fixed